docs/product-oracle-weblogic-installation-and-configuration-guide/src/main/asciidoc/chap-configure.adoc

```diff
@@ -85,122 +58,49 @@ endif::BA[]
 ifdef::DM[]
 ----
 JAVA_OPTIONS="-Dkie.server.jms.queues.response=jms/KIE.SERVER.RESPONSE
- -Dorg.jbpm.designer.perspective=ruleflow
- -Dorg.uberfire.start.method=ejb
- -Dorg.uberfire.domain=OracleDefaultLoginConfiguration
- -Dorg.kie.executor.jms.cf=jms/cf/KIE.EXECUTOR
- -Dorg.kie.executor.jms.queue=jms/KIE.EXECUTOR
+ -Dorg.org.kie.server.domain=OracleDefaultLoginConfiguration
  -Dcom.sun.jersey.server.impl.cdi.lookupExtensionInBeanManager=true"
 ----
 endif::DM[]
 
 [id='_configuring_security_settings']
 == Configuring Security Settings
 
-Several security settings on Oracle WebLogic Server must be set for the Business Central application to work. The following settings enable the container managed authentication mechanisms provided by the WebLogic server:
+Several security settings on Oracle WebLogic Server must be set for the {KIE_SERVER} application to work. The following settings enable the container managed authentication mechanisms provided by the WebLogic server:
 
 . In the WebLogic administrative console, click *Security Realms*.
 . Choose your desired security realm or click *New* to create a new security realm.
 . Navigate to *Users and Groups* -> *Groups* to get to the groups list for your security realm.
 
 ifdef::BA[]
-. Click *New* to create a new group. Create the following new groups: `admin`, `analyst`, `developer`, `manager`, `user`, and `kie-server`. Also create the REST API groups if you will use API. For further information about API roles, see section {URL_DEVELOPMENT_GUIDE}#sect_rest_api[REST API] of _{DEVELOPMENT_GUIDE}_.
+. Click *New* to create a new group. Create the `kie-server` group.
 endif::BA[]
 ifdef::DM[]
-. Click *New* to create a new group. Create the following three new groups: `admin`, `analyst`, and `kie-server`. Also create the REST API groups if you will use API. For further information about API roles, see section {URL_DEVELOPMENT_GUIDE}#sect_rest_api[REST API] of _{DEVELOPMENT_GUIDE}_.
-endif::DM[]
-
-. Click on the *Users* tab and click *New* to create a new user. Provide this new user with a name (for example `business-central-admin`) and a password. Click *OK* to save.
+. Click *New* to create the `kie-server` group.
+. Click the *Users* tab and click *New* to create a new user. Provide this new user with a name (for example `kie-server`) and a password. Click *OK* to save.
```

Comment:
Please change the user name in the example, for example to server-user or anything else. User with kie-server name is not permitted, see important note below.